tools/toolchains/arm.py

```diff
@@ -66,6 +66,9 @@ def __init__(self, target, notify=None, macros=None,
         self.asm = [main_cc] + self.flags['common'] + self.flags['asm']
         self.cc = [main_cc] + self.flags['common'] + self.flags['c']
         self.cppc = [main_cc] + self.flags['common'] + self.flags['c'] + self.flags['cxx']
+        if target.name in ["REALTEK_RTL8195AM"]:
```

Comment:
>  @theotherjimmy We think enum_is_int will added only when target.name is REALTEK_RTL8195AM, So ... may I ask what kind of blocking is this?

Target should not change a configuration like this. Is it portable? If I write an application, switch to this target, enum might have different sizes compare to using different target, but the same application. Or use filesystem (lets say non-volatile memory ) to store something.  Sizes will be different. We care about portability, and obviously this settings it not portable (I assume you receive linker errors if you dont define this flag when linking with a code that has this setting differently).

> Unfortunately, ENUM size has to be 4 bytes for RTL8195. That's why there's target-specific modifications in arm.py/gcc.py/iar.py.

Out of curiosity - How does the library rely on this flag? You use something that compiler takes care of. Enum should be big enough to store all defined values.